src/lib/forms/AccordionField.js

```diff
@@ -9,6 +9,7 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { Field, FastField } from "formik";
 import { Accordion, Container } from "semantic-ui-react";
+import _ from "lodash";
```

Comment:
Can you please import only the func? This will avoid to import the entire `lodash` and take advantage of the [tree shaking](https://webpack.js.org/guides/tree-shaking/):
```suggestion
import _omit from "lodash/omit";
```